src/Orleans/AssemblyLoader/AssemblyLoader.cs

```diff
@@ -12,7 +12,7 @@ internal class AssemblyLoader
         private readonly Dictionary<string, SearchOption> dirEnumArgs;
         private readonly HashSet<AssemblyLoaderPathNameCriterion> pathNameCriteria;
         private readonly HashSet<AssemblyLoaderReflectionCriterion> reflectionCriteria;
-        private readonly TraceLogger logger;
+        private readonly LoggerImpl logger;
```

Comment:
Replaced most LoggerImpl usages with logger.  Was not able to replace them all, without more refactoring.